5-causation/03-matching/notes.qmd

```diff
@@ -30,7 +30,7 @@ If Price and Wolfers had reached an agreement with the NBA that allowed them to
 
 > Assignments of referees to crews are made to balance the experience of referees across games, with groups of three referees working together for only a couple of games before being regrouped. According to the NBA, assignments of refereeing crews to specific (regular season) games is "completely arbitrary" with no thought given to the characteristics of the competing teams. Each referee works 70 to 75 games each year, and no referee is allowed to officiate more than nine games for any team, or referee twice in a city within a fourteen-day period. Although these constraints mean that assignment of refereeing crews to  games is not literally random, the more relevant claim for our approach is that assignment decisions are unrelated to the racial characteristics of either team.[^pricewolfers]
 
-To back up their claim, Price and Wolfers presented covariate balance tables.  The first one gives tests for each of the 12 seasons of data for the null hypothesis that the  number of Black starters is independent of the number of non-Black (described in the paper as "white") referees who officiate its games (which would be true if referees were actually randomly  assigned). The second table examines several other variables to see if they  are significantly associated with the number of non-Black referees, including year, attendance, indicator variables for individual teams.  The table suggests that the number of non-Black referees did vary over time (notice the .00 p-values in the first row of the second table) but most of the other p-values are fairly large, suggesting approximate balance.
+To back up their claim, Price and Wolfers presented covariate balance tables.  The first one presents tests for each of the 12 seasons of data under the null hypothesis that the number of Black starters is independent of the number of non-Black (described in the paper as "white") referees who officiate its games (which would be true if referees were actually randomly  assigned). The second table examines several other variables to see if they  are significantly associated with the number of non-Black referees including year, attendance, and indicator variables for individual teams.  The table suggests that the number of non-Black referees did vary over time (notice the .00 p-values in the first row of the second table) but most of the other p-values are fairly large, suggesting approximate balance.
 
 ![](images/balance_table1_price_wolfers.png){#fig-inflation width="600px"}
 
```

Comment:
Table 2 is also pretty complex, both in layout and terminology. For example, I'm not sure that pointing to the p-values of year labelled "year fixed effect" will convey what that means to students. Do you think this table, too, could be summarized in a more familiar format?